### ISSUE TITLE
Structured data should be included at the root level

### ISSUE THREAD

**Issue by binarylogic** [timberio/timber-js#40]
Timber has evolved it's schema since this library was first designed, it no longer requires a specific shape for events. As such, structured data should be included in the root of the document.

For example:

```
timber.info("Order #1234 placed, total: $500.23", {
  order_placed: {
    id: 1234,
    total: 500.23
  },
});
```

Should send the following document:

```
{
  dt: ...,
  message: "Order #1234 placed, total: $500.23",
  order_placed: {
    id: 1234,
    total: 500.23
  }
}
```

Currently, this is nested under a `context` key and it should not be.

**Comment by steveklebanoff** [comment]
Using TypeScript, I have to do report context in the following way AFAIK.  The type definitions won't allow me to exclude the `context:` key in the second parameter.

```ts
timber.info('Some message', { context: { neptuneTester: 'demo' } });
```

In the timber.io interface, the "context" is double nested:

![image](https://user-images.githubusercontent.com/210914/53917151-c16c3e00-4018-11e9-866a-74378eea737e.png)

Is there any way around this with the current package?

**Comment by binarylogic** [comment]
Hey @steveklebanoff, thanks for letting us know. I'll let @leebenson chime in since he's the author. We plan on having him address this issue this week.

**Comment by leebenson** [comment]
@steveklebanoff, this has now been implemented in [v0.31.0](https://github.com/timberio/timber-js/releases/tag/v0.31.0). Bump whichever NPM package you're using and you should see a much more flexible second parameter.

Per @binarylogic's original example:

```ts
timber.info("Order #1234 placed, total: $500.23", {
  order_placed: {
    id: 1234,
    total: 500.23
  },
});
```

Which now correctly places context at the 'root' level:

<img width="879" alt="Screenshot 2019-03-09 at 13 05 38" src="https://user-images.githubusercontent.com/719722/54071974-2e2b4680-426c-11e9-98db-c1ff94ec8ef8.png">

The Typescript types will feed correctly through to the resolver, too:

```ts
t.info("Order #1234 placed, total: $500.23", {
  order_placed: {
    id: 1234,
    total: 500.23,
  },
}).then(log => {
  console.log("id logged:", log.order_placed.id); // <-- this will full Intellisense
});
```
